cmd/statehash/statehash.go

```diff
@@ -11,6 +11,7 @@ import (
 	"runtime"
 	"strings"
 
+	"github.com/pkg/errors"
```

Comment:
We are trying to get rid of this unsupported package and planing to remove it completely form entire project and you introduce it here :)

Reply:
Fixed